src/resources/elements/primeDesignSystem/ptooltip/ptooltip.ts

```diff
@@ -15,7 +15,12 @@ export class PTooltip {
   private tooltip: Instance;
 
   constructor(private element: Element) {
-    this.tooltip = tippy(this.element, {theme: "prime-design-system"});
+    /** Value found by getting width of 42 'M's of font size 14px. */
+    const maxWidth = 560;
```

Comment:
make it a bindable just in case someone has a really special use case where they want a bigger tooltip